AgentDiscoveries-Backend/src/main/resources/db/migration/V1__add_initial_tables.sql

```diff
@@ -27,6 +27,8 @@ create table regions (
 create table locations (
   location_id int NOT NULL AUTO_INCREMENT PRIMARY KEY,
   site_name varchar(20) NOT NULL,
+  longitude decimal(9,7),

```

Comment:
Why have we added these to V1 rather than a new migration? This will prevent migrations from working on other machines, and e.g. for a production machine would mean wiping the entire database in order to get these columns!